src/api/messages.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @flow
+ * @file i18n messages
+ * @author Box
+ */
+
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
```

Comment:
The API layer should not have any messages. Messages is a UI thing, and the API layer should only be returning codes.

Reply:
I agree with @priyajeet that the API layer probably shouldn't be returning user-facing strings and the consuming component should map the error code/message, instead. That could be handled in a future refactor, though.